test/EndToEndTests/Services/ODataSimplifiedService/ODataSimplifiedDataSource.cs

```diff
@@ -171,9 +171,12 @@ protected override IEdmModel CreateModel()
         protected override void ConfigureContainer(IContainerBuilder builder)
         {
             base.ConfigureContainer(builder);
-            builder.AddServicePrototype(new ODataSimplifiedOptions()
+            var writerSettings = new ODataMessageWriterSettings();

```

Comment:
Shouldn't we at least rename "ODataSimplifiedService" and "ODataSimplifiedDataSource" to avoid confusion? Or move the tests in here to the respective `ODataMessageWriterSettings`, `ODataMessageReaderSettings`, and `ODataUriParserSettings` tests file?

Reply:
Ooh. I didn't want to make too many changes to the E2E tests because we're having challenges running them in CI. Just wanted to make the minimum changes to ensure we can still build and run them.

Reply:
If you choose not to rename now, the other option will be to maintain a list of what you're deferred for later in a work item.

Reply:
I've created an issue here: https://github.com/OData/odata.net/issues/2878